Mathlib/Data/Nat/Size.lean

```diff
@@ -35,26 +31,15 @@ theorem shiftl'_tt_eq_mul_pow (m) : ∀ n, shiftl' true m n + 1 = (m + 1) * 2 ^
 
 end
 
-theorem one_shiftl (n) : shiftl 1 n = 2 ^ n :=
-  (shiftl_eq_mul_pow _ _).trans (Nat.one_mul _)
-#align nat.one_shiftl Nat.one_shiftl
```

Comment:
Where did this lemma go?

Reply:
The same lemma with the exact same name exists in ```Std.Data.Nat.Lemmas```.

Reply:
Can you leave behind the `#align`?

Reply:
It looks like you've swapped the order of the lemmas here; can you put them back in the same order to minimize the diff?